src/EventStore.Common/Utils/ContainerizedEnvironment.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace EventStore.Common.Utils;
+
+public static class ContainerizedEnvironment {
+
+	public const int ContainerReaderThreadCount = 4;
```

Comment:
no need to reapeat the word 'Container' in these names since it's included in the context (ContainerizedEnvironment class)